src/pages/index.js

```diff
@@ -31,56 +81,73 @@ const popupEditProfileForm = new PopupWithForm(
   ".modal_profile-edit",
   handleProfileEditSubmit
 );
-const gallerySection = new Section(
-  { items: initialCards, renderer: cardRenderer },
-  cardGalleryEL
+const popupAvatarForm = new PopupWithForm(
+  ".modal_change-avatar",
+  handleAvatarFormSubmit
+);
+////////////////////////////////////////////////////////////
+///////////////////////   API    ///////////////////////////
+////////////////////////////////////////////////////////////
+const apiCall = new Api(
+  "https://around-api.en.tripleten-services.com/v1",
+  serverToken,
+  apiFunctionObj
 );
-
-//////////////////////////  functions  //////////////////////////
+////////////////////////////////////////////////////////////
+///////////////////////   FUNCTIONS    /////////////////////
+////////////////////////////////////////////////////////////
+function handleDeleteFormSubmit(data) {
+  data.cardElement.remove();
```

Comment:
You need to **delete** cards only in block `then` after a **successful** response from the server so that if there is an error, the card should stay in the `DOM`.